File: demo/java-demo/src/test/java/com/alibaba/demo/association/SellerServiceTest.java
package com.alibaba.demo.association;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SellerServiceTest {

    private SellerService sellerService = new SellerService();

    @Test
    void should_sell_sandwich() {
        assertEquals("Fake-Sandwich-Cooker & Faked-Sandwich", sellerService.sellSandwich());
    }

    @Test
    void should_sell_hamburger() {
        assertEquals("Real-Hamburger-Cooker & Real-Hamburger", sellerService.sellHamburger());
    }

}

File: demo/kotlin-demo/src/test/kotlin/com/alibaba/demo/association/SellerServiceTest.kt
package com.alibaba.demo.association

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class SellerServiceTest {

    private val sellerService = SellerService()

    @Test
    fun should_sell_sandwich() {
        assertEquals("Fake-Sandwich-Cooker & Faked-Sandwich", sellerService.sellSandwich())
    }

    @Test
    fun should_sell_hamburger() {
        assertEquals("Real-Hamburger-Cooker & Real-Hamburger", sellerService.sellHamburger())
    }
}
